### PR TITLE
Docs: Update Grafana-Managed Recording Rule docs

### PR DESCRIPTION
**Pull request by alexweav**
**What is this feature?**

Adds several clarifications/missing info from the GMRR docs.

**Which issue(s) does this PR fix?**:

n/a

**Special notes for your reviewer:**

Please check that:
- [x] It works as expected from a user's perspective.
- [x] If this is a pre-GA feature, it is behind a feature toggle.
- [x] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/contribute/release-notes/#how-to-determine-if-content-belongs-in-whats-new), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/contribute/release-notes/) doc.

### CODE REVIEW

File: docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md [hunk 1]
```diff
@@ -33,13 +33,9 @@ refs:
 
 Recording rules calculate frequently needed expressions or computationally expensive expressions in advance and save the result as a new set of time series. Querying this new time series is faster, especially for dashboards since they query the same expression every time the dashboards refresh.
 
-For more information on recording rules in Prometheus, refer to [Defining recording rules in Prometheus](https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/).
-
-Recording rules are run as instant rules, which means that they run every 10s. To overwrite this configuration, update the min_interval in your custom configuration file.
```
Comment (alexweav) [inline docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md:38]: Drive-by, this line wasn't correct. Also the first bit only applies to datasource managed, so moved it there.

File: docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md [hunk 2]
```diff
@@ -33,13 +33,9 @@ refs:
...
+Grafana-Managed Recording Rules offer the same Prometheus-like semantics, but allow you to query any Grafana datasource supported by alerting. In addition to the above, you can use this to import and map data from other datasources into Prometheus.
 
-[min_interval](ref:configure-grafana) sets the minimum interval to enforce between rule evaluations. The default value is 10s which equals the scheduler interval. Rules will be adjusted if they are less than this value or if they are not multiple of the scheduler interval (10s). Higher values can help with resource management as fewer evaluations are scheduled over time.
```
Comment (alexweav) [inline docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md:40]: Only applies to GMRR, so moved it to that section.

Also this is extremely advanced, so i moved it to the bottom. I have never seen a use case for configuring this.

File: docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md [hunk 3]
```diff
@@ -139,24 +152,20 @@ It does not support absolute time ranges: `2021-12-02 00:00:00 to 2021-12-05 23:
...
-
-   You can only add one recovery threshold in a query and it must be the recording rule output.
-
-4. Click **Set as recording rule output** on the query or expression you want to set as your rule output.
+3. Click **Set as recording rule output** on the query or expression you want to set as your rule output.
 
 #### Set evaluation behavior
 
-Use recording rule evaluation to determine how frequently a recording rule should be evaluated and how quickly it should change its state.
+Use recording rule evaluation to determine how frequently a recording rule should be evaluated.
```
Comment (alexweav) [inline docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md:159]: References to state and pending period (which don't exist in gmrr) removed

File: docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md [hunk 4]
```diff
@@ -139,24 +152,20 @@ It does not support absolute time ranges: `2021-12-02 00:00:00 to 2021-12-05 23:
...
-To do this, you need to make sure that your recording rule is in the right evaluation group and set a pending period time that works best for your use case.
+To do this, you need to make sure that your recording rule is in the right evaluation group with an interval that works best for your use case.
 
 1. Select a folder or click **+ New folder**.
 1. Select an evaluation group or click **+ New evaluation group**.
 
    If you are creating a new evaluation group, specify the interval for the group.
 
-   All rules within the same group are evaluated concurrently over the same time interval.
+   All rules within the same group are evaluated concurrently over the same time interval. Every recording rule in a group will use the same evaluation time, meaning that all queries from the same group will always be aligned with each other.
```
Comment (brendamuir) [inline docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md:168]:
```suggestion
   All rules within the same group are evaluated concurrently over the same time interval. Every recording rule in a group uses the same evaluation time, meaning that all queries from the same group are always aligned with each other.
```

File: docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md [hunk 5]
```diff
@@ -169,3 +178,12 @@ To do this, you need to make sure that your recording rule is in the right evalu
 1. Add custom labels selecting existing key-value pairs from the drop down, or add new labels by entering the new key or value.
 
 1. Click **Save rule** to save the rule or **Save rule and exit** to save the rule and go back to the Alerting page.
+
+#### Advanced Configuration
+
+[min_interval](ref:configure-grafana) sets the minimum interval to enforce between rule evaluations. The default value is 10s which equals the scheduler interval. Rules will be adjusted if they are less than this value or if they are not multiple of the scheduler interval (10s). Higher values can help with resource management as fewer evaluations are scheduled over time.
```
Comment (brendamuir) [inline docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md:184]:
```suggestion
[min_interval](ref:configure-grafana) sets the minimum interval to enforce between rule evaluations. The default value is 10s which equals the scheduler interval. Rules are adjusted if they are less than this value or if they are not multiple of the scheduler interval (10s). Higher values can help with resource management as fewer evaluations are scheduled over time.
```

File: docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md [hunk 6]
```diff
@@ -169,3 +178,12 @@ To do this, you need to make sure that your recording rule is in the right evalu
...
+
+This setting has precedence over each individual rule frequency. If a rule frequency is lower than this value, then this value is enforced.
+
+This setting applies to both Grafana-managed alerting and recording rules.
```
Comment (brendamuir) [inline docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md:188]:
```suggestion
This setting applies to both Grafana-managed alert and recording rules.
```

**Review by brendamuir — APPROVED** [review]

File: docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md [hunk 7]
```diff
@@ -139,24 +152,20 @@ It does not support absolute time ranges: `2021-12-02 00:00:00 to 2021-12-05 23:
 
    b. Click **Preview** to verify that the expression is successful.
 
-3. To add a recovery threshold, turn the **Custom recovery threshold** toggle on and fill in a value for when your recording rule should stop meeting the condition.
-
-   You can only add one recovery threshold in a query and it must be the recording rule output.
-
-4. Click **Set as recording rule output** on the query or expression you want to set as your rule output.
+3. Click **Set as recording rule output** on the query or expression you want to set as your rule output.
```
[the remark this answers is not in the document]
Comment (alexweav) [reply docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md:155]: Hmm, the linter didn't like this one for some reason. Running the linter undid this
